constants/rewards.ts

```diff
@@ -47,22 +47,38 @@ export const WEEKLY_UMA_REWARDS: { [key: string]: any[] } = {
       endDate: Date.UTC(2020, 9, 1, 0, 0, 0),
     },
   ], // uUSDrBTC-DEC
+  "0x90f802c7e8fb5d40b0de583e34c065a3bd2020d8": [
+    {
+      token: "UMA",
+      count: 15000,
```

Comment:
It'd be helpful to add a comment here about how this is an approximation based on expected dev mining allocation

Reply:
Done